beacon-chain/forkchoice/protoarray/optimistic_sync_test.go

```diff
@@ -368,6 +369,7 @@ func TestSetOptimisticToInvalid(t *testing.T) {
 			3,
 			4,
 			8,
+			[][32]byte{[32]byte{'j'}},
```

Comment:
Do you want to spend more time to make these tests have more entropies? The `doubly-linked-tree` ones are returning more roots than just one

Reply:
Talked offline. This is much more involvedd. We'll address this in the next PR